src/SFA.DAS.EmployerIncentives.ValueObjects/Apprenticeship.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.ValueObjects
+{
+    public class Apprenticeship
```

Comment:
Can this inherit from ValueObject to enforce the immutability based on it's data?